jobs/JGLOBAL_AERO_ANALYSIS_FINALIZE

```diff
@@ -8,25 +8,17 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "aeroanlfinal" -c "base aeroanl aeroan
 ##############################################
 # Set variables used in the script
 ##############################################
-# shellcheck disable=SC2153
-GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
-gPDY=${GDATE:0:8}
-gcyc=${GDATE:8:2}
-GDUMP="gdas"
-
 
 ##############################################
 # Begin JOB SPECIFIC work
 ##############################################
 
 # Generate COM variables from templates
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS COM_CHEM_ANALYSIS
-
-RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
-    COM_CHEM_ANALYSIS_PREV:COM_CHEM_ANALYSIS_TMPL \
-    COM_ATMOS_RESTART_PREV:COM_ATMOS_RESTART_TMPL
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMOUT_CHEM_ANALYSIS:COM_CHEM_ANALYSIS_TMPL \
+    COMOUT_ATMOS_RESTART:COM_ATMOS_RESTART_TMPL
 
-mkdir -m 775 -p "${COM_CHEM_ANALYSIS}"
+mkdir -p "${COMOUT_CHEM_ANALYSIS}"
```

Comment:
Does `COMOUT_ATMOS_RESTART` need to be created as well?

Reply:
No (or at least I don't think so), because now the fcst job copies the RESTART files from COM_CHEM_ANALYSIS if appropriate like how it does with the surface files

Reply:
Is `COMOUT_ATMOS_RESTART` still needed at all then?

Reply:
oh I'm dumb, the FileHandler creates it in Jinja, see: https://github.com/NOAA-EMC/global-workflow/blob/3936a79e54e57aa0c1b258a1487ebef8e88b1c68/parm/gdas/aero_finalize_variational.yaml.j2#L12(I guess it should be moved to the j-job, what do you think?)

The fcst job does copy, but it is copying `aeroanl` files in `COM_ATMOS_RESTART` and not from `COM_CHEM_ANALYSIS`

Reply:
@CoryMartin-NOAA Does this still need to be addressed?

Reply:
I'll defer to @WalterKolczynski-NOAA on if `COMOUT_ATMOS_RESTART` needs created here, in the YAML with FileHandler (as it is currently), or both

Reply:
Sorry, I've been out-of-sorts the last week. If the yaml is creating the directory before it is needed, that is fine. (But why isn't it also doing the chem analysis directory then?)

Reply:
Got it. I think the latest commit should resolve this. There's no need for it to be created in the j-job script, the YAML will create it before copying files to it